networks/suzuka/suzuka-client/src/tests/mod.rs

```diff
@@ -0,0 +1,138 @@
+use anyhow::{Context, Result};
+use crate::{
+    coin_client::CoinClient,
+    rest_client::{Client, FaucetClient},
+    types::LocalAccount,
+};
+use once_cell::sync::Lazy;
+use std::str::FromStr;
+use url::Url;
+
+static MONZA_CONFIG : Lazy<movement_execution_util::config::Config> = Lazy::new(|| {
```

Comment:
`
```suggestion
static SUZUKA_CONFIG : Lazy<suzuka_execution_util::config::Config> = Lazy::new(|| {
```

Reply:
Done that, except that this configuration is not currently specific to Suzuka.